includes/wikia/Wikia.php

```diff
@@ -1314,11 +1314,11 @@ static public function setUpMemcachePurge( WebRequest $request, $user ) {
 	 * @return bool true
 	 */
 	static public function onUserMailerSend( &$to ) {
-		foreach ( $to as $u ) {
-			if ( $u instanceof MailAddress && $u->name != '' ) {
-				$u->name = '';
-			}
-		}
+//		foreach ( $to as $u ) {
```

Comment:
I guess this is commented on purpose?

Reply:
Yup, I don't know what is the original purpose of this code but I need the username for GDPR email.